### PR TITLE
Add referer to default_media soft assert

### PR DESCRIPTION
**Pull request by orangejenny**
Followup for https://github.com/dimagi/commcare-hq/commit/dfb338159d8e4f942ba5aac15bba777c48048f3a - I've gotten a few of these but don't know what app/domain they're coming from, which makes them pretty useless.

@dannyroberts / @czue

### CODE REVIEW

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -1500,8 +1500,13 @@ def _assert_unexpected_default_media_call(self, media_attr):
         assert media_attr in ('media_image', 'media_audio')
         media = getattr(self, media_attr)
         if isinstance(media, dict) and list(media) == ['default']:
+            from corehq.util.view_utils import get_request
+            request = get_request()
+            url = ''
+            if request:
+                url = request.META.get('HTTP_REFERER')
```
Comment (czue) [inline corehq/apps/app_manager/models.py]: TIL this misspelling is [in the HTTP standard](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.36) :man_facepalming:

**Review by czue — APPROVED** [review]

File: corehq/apps/app_manager/models.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/app_manager/models.py]: aww, that's a shame

Comment (orangejenny) [reply corehq/apps/app_manager/models.py]: Unrelated, a lot of emoticons just show up as boxes in my email, but that particular one shows as a box and a male symbol, which always confuses me for half a second:

screen shot 2018-03-30 at 10 59 23 am

Comment (czue) [reply corehq/apps/app_manager/models.py]: I would react to this with a facepalm emoji but I fear that would only exacerbate the situation

**Pull request merged**